### PR TITLE
Adds multiple payment options for invoices

### PR DESCRIPTION
**Pull request by zigge-zagge**
API proposal (Tenant API) for multiple payment options.

See it here: http://developer.kivra.com/jz-multiple-payment-options/index.html#operation/Send%20content

Issue: [STS-399](https://kivra.atlassian.net/browse/STS-399)
Parent issue: [STS-370](https://kivra.atlassian.net/browse/STS-370)

Related PR: https://github.com/kivra/developer.kivra.com/pull/43

## Questions

* What should the max size be for a payment option icon? I've written 100 kB but that's just something I've made up.
* I have changed from `total_owed` to `amount` since the former doesn't make sense here. Is that okay?

## Notes
* I have changed from `subject` to `title` which I think is a better name.
* See #43 for 'no thanks' info

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -1163,7 +1163,7 @@ paths:
       tags:
         - "Tenant API - Content"
       summary: Send content to a recipient (user or company)
-      operationId: Send content
+      operationId: post_v1_tenant_key_content
```
Comment (zigge-zagge) [inline swagger.yml:1166]: I'll change it back. We can update all of them later instead if we'd like.

```suggestion
      operationId: Send content
```

**Review by chsukivra — APPROVED** [review]

**Review by paolokivra — APPROVED** [review]
👍

File: swagger.yml [hunk 2]
```diff
@@ -1925,6 +1962,86 @@ components:
...
+              amount:
+                type: string
+                format: float
+                example: "123.50"
+                description: The payment amount for this option. A positive number.
+              type:
+                type: string
+                description: Type of format for the reference
+                example: "SE_OCR"
+                enum: ["SE_OCR", "TENANT_REF"]
```
Comment (cdyb-kivra) [inline swagger.yml:2031]: What about Finland?

**Comment by cdyb-kivra** [comment]
Malotte and I have asked Malin to put which parts of a payment should be allowed to vary on the agenda for tomorrow's typeforce meeting, since we feel that several points are not obvious. Perhaps this PR could be an input to that discussion?

**Pull request closed without merging**